scripts/index.js

```diff
@@ -43,6 +36,12 @@ function aboutPageGenerator() {
   about.appendChild(div);
 }
 
+Project.loadAll = function(data) {
+  data.forEach(function (object) {
+    Project.all.push(new Project(object));
+  })
+};
+
 projectData.forEach(function(projectObject) {
   projects.push(new Project(projectObject));
```

Comment:
Here you will also want to push the new Projects into Project.all, instead of pushing into projects.